Linux-mac-install.md

```diff
@@ -9,15 +9,15 @@ While these instructions contain commands to install both SQLSRV and PDO_SQLSRV
 
 ## Contents of this page
 
-- [Installing the drivers on Ubuntu 16.04, 18.04, and 20.04](#installing-the-drivers-on-ubuntu-1604-1804-and-2004)
+- [Installing the drivers on Ubuntu 18.04, 20.04 and 21.04](#installing-the-drivers-on-ubuntu-1804-2004-and-2104)
 - [Installing the drivers with PHP-FPM on Ubuntu](#installing-the-drivers-with-php-fpm-on-ubuntu)
 - [Installing the drivers on Red Hat 7 and 8](#installing-the-drivers-on-red-hat-7-and-8)
 - [Installing the drivers on Debian 9 and 10](#installing-the-drivers-on-debian-9-and-10)
 - [Installing the drivers on Suse 12 and 15](#installing-the-drivers-on-suse-12-and-15)
-- [Installing the drivers on Alpine 3.11 and 3.12](#installing-the-drivers-on-alpine-311-and-312)
+- [Installing the drivers on Alpine 3.11, 3.12 and 3.13](#installing-the-drivers-on-alpine-311-312-and-313)
 - [Installing the drivers on macOS Mojave, Catalina and Big Sur](#installing-the-drivers-on-macos-mojave-catalina-and-big-sur)
```

Comment:
We should remove the versions from the heading on these lines so that the anchor stays consistent.  There isn't really a need to have the versions in the headings here. Every time we change the version/heading/anchor, we partially invalidate any previous URLs that referenced that anchor (the URL will just go to the top of the page instead of the intended anchor).

Reply:
Ok thanks @David-Engel  I will fix these

Reply:
@David-Engel do you have any more comment? I've also made some minor changes to the Change Log